index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no" />
+
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>TLNT - Changing the casting industry</title>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' *.tlnt.io tlnt.io; child-src 'none'; frame-src www.youtube.com; font-src fonts.gstatic.com; style-src 'unsafe-inline' *.tlnt.io tlnt.io">
+    <link rel="stylesheet" href="dist/styles-min.css">
+
+    <!-- Favicon and theme -->
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+    <link rel="manifest" href="/site.webmanifest">
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#0552e2">
+    <meta name="msapplication-TileColor" content="#0552e2">
+    <meta name="theme-color" content="#ffffff">
+
+    <meta property="og:image" content="/og-image.png">
+    <meta property="og:image:height" content="521">
+    <meta property="og:image:width" content="994">
+    <meta property="og:description" content="Changing the casting industry by creating a thriving and bustling community where talents &amp; clients are fully in control of their own personal profile data and financial transactions.">
+    <meta property="og:title" content="TLNT - Changing the casting industry">
+    <meta property="og:url" content="http://tlnt.io">
+  </head>
+  <body>
+
+    <nav class="navbar navbar-expand-lg navbar-light">
+      <div class="container">
+        <a class="navbar-brand" href="#">
+          <div class="logoWrap">
+            <img src="assets/images/logo-header.svg" alt="Logo TLNT">
```

Comment:
All content locations should start with /

Reply:
☑️